shenyu-integrated-test/shenyu-integrated-test-common/src/main/java/org/apache/shenyu/integratedtest/common/helper/HttpHelper.java

```diff
@@ -264,4 +265,25 @@ public Response getHttpService(final String url, final Map<String, Object> heade
         Request request = requestBuilder.build();
         return client.newCall(request).execute();
     }
+
+    /**
+     * Send a get http request to shenyu gateway .
+     *
+     * @param <S>     response type
+     * @param headers headers
+     * @param path    path
+     * @param type    type of response passed to {@link Gson#fromJson(String, Type)}
+     * @return response from gateway
+     * @throws IOException IO exception
+     */
+    public <S> S getHttpService(final String path, final Map<String, Object> headers, final Type type) throws IOException {
+        Response response = getHttpService(path, headers);
+        String respBody = Objects.requireNonNull(response.body()).string();
+        LOG.info("testMetricsPluginFromGateway({}) resp({})", path, respBody);
```

Comment:
Pls modify the log, this method should be a common util.

Reply:
ok